pkgs/data/fonts/siji/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchgit }:
+
+stdenv.mkDerivation rec {
+  name = "siji-${date}";
+  date = "2016-05-14";
```

Comment:
Ok, I think 14 is fine, as that's a day the commit was made (in the author's timezone)